specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3623,7 +3583,7 @@
       "type": "object",
       "description": "Request parameters to transfer billing subscription.",
       "properties": {
-        "destinationInvoiceSectionName": {
+        "destinationInvoiceSectionId": {
           "type": "string",
           "description": "The destination invoiceSectionName."
```

Comment:
Change "invoiceSectionName" to "invoice section id" to be consistent.

Reply:
Sure!